sass/_core.scss

```diff
@@ -23,6 +23,8 @@
 ///   and provides specific tools for maintaining those values
 /// @group core
 
+$_a_strict: false !default;
```

Comment:
```suggestion
$strict-mode: false !default;
```
or perhaps
```suggestion
$error-on-missing-keys: false !default;
```
to make it really obvious? I'll add some inline documentation for the option for Sassdoc to pick up. Because of how the code is handling locating aliases, adding a `/` as a terminating character is actually trivial, I'll add it in tomorrow and write a test.

Reply:
I like the more explicit `$error-on-missing-keys`, but don't have permissions to commit on your branch. :)

Reply:
I have no idea how this feature works and I've mucked something up. Renaming it on my branch so you can pull the corrected code.